### PR TITLE
[DEVOPS-67] disable burst limiting of logs

### PR DESCRIPTION
**Pull request by domenkozar**
This also includes two fixes already deployed to testnet. Need to get them in ASAP for any further deploys.

### CODE REVIEW

File: deployments/cardano-prod.nix [hunk 1]
```diff
@@ -6,7 +6,13 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["prod"];
+    services.dd-agent.tags = ["env:production"];
+
+    # DEVOPS-64: disable log bursting
+    services.journald.rateLimitBurst = 0;
```
Comment (deepfire) [inline deployments/cardano-prod.nix]: Hmm, AIUI, without this option, `journald` would trim an excessively chatty stream of similar messages to 1 summary message per 30s.

This change effectively disables that trimming, forcing all messages to appear in the log.

How would that help?

File: deployments/cardano.nix [hunk 2]
```diff
@@ -86,8 +84,13 @@ in
...
       # TODO: generalize with node generation
-      (genAttrs' (range 0 5) (key: "nodeip${toString key}") (name: { region = "eu-central-1"; inherit accessKeyId; })) //
-      #(genAttrs' (range 7 13) (key: "nodeip${toString key}") (name: { region = "us-west-1"; inherit accessKeyId; })) //
+      (genAttrs' (range 0 1) (key: "nodeip${toString key}") (name: { region = "eu-central-1"; inherit accessKeyId; })) //
+      (genAttrs' (range 2 3) (key: "nodeip${toString key}") (name: { region = "eu-west-1"; inherit accessKeyId; })) //
+      (genAttrs' (range 4 5) (key: "nodeip${toString key}") (name: { region = "eu-west-2"; inherit accessKeyId; })) //
+      (genAttrs' (range 6 7) (key: "nodeip${toString key}") (name: { region = "ap-southeast-1"; inherit accessKeyId; })) //
+      (genAttrs' (range 8 9) (key: "nodeip${toString key}") (name: { region = "ap-southeast-2"; inherit accessKeyId; })) //
+      (genAttrs' (range 10 11) (key: "nodeip${toString key}") (name: { region = "ap-northeast-1"; inherit accessKeyId; })) //
+      (genAttrs' (range 12 13) (key: "nodeip${toString key}") (name: { region = "ap-northeast-2"; inherit accessKeyId; })) //
```
Comment (deepfire) [inline deployments/cardano.nix]: This effectively makes `cardano.nix` to correspond to the production version, right?

Comment (domenkozar) [inline deployments/cardano.nix]: Correct, as it should be.

File: deployments/cardano-prod.nix (hunk 1 above)
Comment (domenkozar) [inline deployments/cardano-prod.nix]: Currently, logs are trimmed, so we could be missing important information in log events. This helps us collect all logs.

**Pull request merged**